worksheet/index.qmd

```diff
@@ -0,0 +1,131 @@
+---
+format: typst
+---
+
+```{=typst}
+#set page(
+  flipped: true, 
+  columns: 2, 
+  margin: (x: 0.5in, y: 1in),
+  numbering: none
+)
+```
+
+## Writing your first documentation PR
+
+The goal of this worksheet is to guide you through the steps needed to successfully create a documentation PR (pull request). Empty spaces have been placed on this sheet for your notes.
+
+### 0. Get ready
+
+1. `install.packages("pak")`
+
+1. `pak::pak("devtools")`
+
+1. .Set up your GitHub personal access token.
+    - `usethis.r-lib.org` -> Articles -> Managing Git(Hub) Credentials
+
+1. Call `usethis::git_sitrep()` and check that:
+    - Your name and email address appears in "Git config (global)"
+    - Your GitHub user name is found under "Github"
+
+```{=typst}
+#v(1.5in)
+```
+
+### 1. Find and claim an issue
+
+Browse the Post-Its, looking for something of interest. If you've never done a PR before, we recommend that you start with a documentation issue since there are slightly fewer moving parts. We also encouraged you to team up with a (new) friend and tackle something together!
+
+Once you've picked a Post-it, open the issue and read the details in full. At this point, you might discover the issue is outside your wheelhouse; if so, no problem, just return the Post-it to the wall and try again.
+
+```{=typst}
+#colbreak()
+#v(3.5in)
+```
+
+### 2. Get the source locally
+
+* **Fork** and clone the repository using this syntax: 
+    - `usethis::create_from_github("{org}/{repo}")` 
```

Comment:
How hard would it to be to make this fill-in-the-blanks too? I think ideally it would be a longer line with org/repo in pale grey text, but I appreciate that might be more work than it's worth.

Reply:
greying out text was a little hard, tried empty spacing instead